composer.json

```
@@ -156,14 +160,19 @@
         "symfony/ux-live-component": "^2.15",
+        "symfony/ux-toggle-password": "^2.24",
         "symfony/validator": "^7.1",
         "symfony/vonage-notifier": "^7.1",
+        "symfony/webhook": "^7.1",
         "symfony/webpack-encore-bundle": "^2.0",
         "symfony/workflow": "^7.1",
         "symfony/yaml": "^7.1",
         "symfony/yunpian-notifier": "^7.1",
         "symfony/zulip-notifier": "^7.1",
         "symfonycasts/dynamic-forms": "^0.1.2",
+        "symfonycasts/reset-password-bundle": "^1.23",
```

Biome (1.9.4)

[error] The key symfonycasts/reset-password-bundle was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)